menstruation/jobs.py

```diff
@@ -0,0 +1,92 @@
+import logging
+from json import JSONDecodeError
+from random import randint
+from time import sleep
+
+from emoji import emojize
+from telegram.error import Unauthorized
+
+from menstruation import config
+from telegram.ext import CallbackContext, JobQueue, Job
+
+from menstruation.handlers import send_menu
+from menstruation.query import Query
+
+user_db = config.user_db
+job_queue = None
+
+
+def startup_message(context: CallbackContext):
+    for moderator in config.moderators:
+        try:
+            context.bot.send_message(moderator, emojize("Server wurde gestartet :robot_face:"))
+        except Unauthorized:
+            logging.exception(f"Moderator: {moderator}, has blocked the Bot.")
+        except Exception as err:
+            logging.exception(f"Error in startup_message: {err}")
+
+
+def add_subscriber(user_id: str):
+    logging.debug(f"Added subscriber: {user_id}")
+    job_queue.run_daily(
+        notify_subscriber,
+        config.notification_time,
+        days=(0, 1, 2, 3, 4),
+        name=user_id,
+    )
+
+
+def remove_subscriber(user_id: str):
+    logging.debug(f"Removed subscriber: {user_id}")
+    for job in job_queue.get_jobs_by_name(user_id):
+        logging.debug(f"Job: {job.name}, {job.enabled}, {job.removed}")
```

Comment:
boolean blindness: better log `enabled: {job.enabled}, removed: {job.removed}`

Reply:
That should not be there xD